server/src/main/java/com/cloud/network/NetworkModelImpl.java

```diff
@@ -2207,14 +2207,8 @@ public boolean isNetworkInlineMode(Network network) {
 
     @Override
     public void checkIp6Parameters(String startIPv6, String endIPv6, String ip6Gateway, String ip6Cidr) throws InvalidParameterValueException {
-        if (!NetUtils.isValidIp6(startIPv6)) {
-            throw new InvalidParameterValueException("Invalid format for the startIPv6 parameter");
-        }
-        if (!NetUtils.isValidIp6(endIPv6)) {
-            throw new InvalidParameterValueException("Invalid format for the endIPv6 parameter");
-        }
 
-        if (!(ip6Gateway != null && ip6Cidr != null)) {
+        if (org.apache.commons.lang3.StringUtils.isBlank(ip6Gateway) || org.apache.commons.lang3.StringUtils.isBlank(ip6Cidr)) {
```

Comment:
can you encapsulate this in our local proxy to the StringUtils, so migration efforts will be located in a single file?

Reply:
Fixed @DaanHoogland, code is now using the  `com.cloud.utils.StringUtils.isBlank`. Thanks!